eyeshade/workers/reports.js

```diff
@@ -927,9 +924,10 @@ const findEligPublishers = async (debug, runtime, publishers) => {
  **/
 const prepareReferralPayout = async (debug, runtime, authority, reportId, thresholdProbi) => {
   const owners = runtime.database.get('owners', debug)
+  const publishers = runtime.database.get('publishers', debug)
 
   const statements = await referralStatement(debug, runtime, undefined, true)
-  console.log(statements)
+  console.log(JSON.stringify(statements))
```

Comment:
could e put a comment before this, perhaps something like:

        // NB: REMOVE THIS SOON!